### DISCUSSION TITLE
Peculiar flux/width concentration

### DISCUSSION THREAD

**Discussion by 0xCoto** [chime-frb-open-data/community#27] [Q&A] [answered]
I have just parsed the entire catalog to [FRBSTATS](https://www.herta-experiment.org/frbstats/) (the repeaters will soon be sorted to present more accurate information in that particular regard). Other than that, everything else should be highly accurate.

Here's what CHIME's contribution to the distribution of FRBs looks like...

## Before catalog release (this already consists of more (>2x) CHIME FRBs!):

![dec_ra](https://user-images.githubusercontent.com/25392776/121796505-2b291880-cc22-11eb-8b41-d95d19bd18f2.png)

<img width="1440" alt="long_lat" src="https://user-images.githubusercontent.com/25392776/121796463-d2f21680-cc21-11eb-9828-b17c07a10525.png">

## After catalog import:

![dec_ra (1)](https://user-images.githubusercontent.com/25392776/121796523-3bd98e80-cc22-11eb-8d9e-895ea52a2835.png)

<img width="1440" alt="long_lat (1)" src="https://user-images.githubusercontent.com/25392776/121796489-0df44a00-cc22-11eb-9a0b-b898d16f5951.png">

Very nice to see such a huge increase in the detected population of FRBs. This rich dataset has just brought FRBSTATS to an event coverage of **95.9%**.

---

### One thing I wasn't quite sure about is this:

![CHIME_flux_width](https://user-images.githubusercontent.com/25392776/121796407-6545ea80-cc21-11eb-8e99-dbd7d8ccc14f.png)

I've filtered the plot to solely present the CHIME FRBs, and it appears as if there is some kind of unexpectedly odd concentration along both axes, as if some kind of unusual rounding has taken place.

Any ideas on what the origin of this peculiar distribution could be?

**Comment by zpleunis** [comment, marked as answer]
This looks great @0xCoto, thanks for your efforts! I am sure the community will benefit from this.

The concentration of values that you point out is expected: the total burst width that you are plotting (`bc_width` in the catalog) is by definition a multiple of the sampling time of the data, which is 0.98304 ms. The errors on the peak fluxes are in the range 0.1--20.0 Jy and the peak flux values are rounded to give only the significant numbers in the results, which results in the maximum precision of the values being a tenth of a Jy.
